packages/fluentui/docs/src/components/ComponentDoc/ExampleSection.tsx

```diff
@@ -18,15 +18,9 @@ const ExampleSection: React.FC<ExampleSectionProps> = ({ title, children }) => (
     <Header as="h2" align="center" styles={headerStyle} className="no-anchor">
       {title}
     </Header>
-    <Grid
-      variables={{ gridGap: '2rem' }}
-      styles={{
-        gridTemplateColumns: 'minmax(550px, 1fr)',
```

Comment:
And this breaks also:

![image](https://user-images.githubusercontent.com/14183168/84010299-c551e300-a974-11ea-94a2-080ee02f243d.png)

![image](https://user-images.githubusercontent.com/14183168/84010339-cf73e180-a974-11ea-902f-2517a217cd75.png)

Reply:
The problem seems to be an issue with `Toolbar` component when inside a container with `display: flex`. To temporary fix it I changed the example itself, but I guess it is a toolbar issue. Right @pompomon @layershifter @jurokapsiar ?

Reply:
Fixed as agreed stretching the container for now, but I think we might still wanna to report the issue for the toolbar?

Reply:
> Fixed as agreed stretching the container for now, but I think we might still wanna to report the issue for the toolbar?

Yup, let's do it